### PR TITLE
Raises the price of skeleton keys to 1,666

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Raises the price of skeleton keys by 114.42%, making them mildly more expensive without effectively eliminating from being obtained at all, like https://github.com/tgstation/tgstation/pull/66239 was stupidly doing by raising the price by 900.9% and was just a grudge PR with a decent incentive but horrendous balancing.

## Why It's Good For The Game

Makes skeleton keys just *a bit* more scarce, requiring now twice the price than before.
You'll be able to get them just fine by mining like normal, being a similar price to the Mining Conscription Kit (1500 points).
Miners will have to mine twice as much to get them, giving a good tradeoff between work and loot.

## Changelog
:cl:
balance: Skeleton keys now cost 1,666 mining points. Get to work, miners.
/:cl:

### CODE REVIEW

**Comment by Livrah** [comment]
Good ol' "make shitpost nerf PR so others will do a work for me".

**Comment by tralezab** [comment]
> Good ol' "make shitpost nerf PR so others will do a work for me".

wdym work it's a single line

**Comment by tralezab** [comment]
can i also ask before we do this song and dance again if there is actually a problem of miners not bringing back enough ore

**Comment by SgtHunk** [comment]
I'd say enough of an issue where incentive of price increase is worth it.
Part of it depends on how far mining research has gone and how eager are miners to come back.
If a miner were to rush a tendril, their steps would be to gain just enough ores for around 800~ points , this PR would make them bring twice as much ores worth. At the very least, it'll help with early-game material amounts.

**Comment by rasepretep** [comment]
The previous 7777 PR was clearly either bait or grudge PR. Based on what Fikou said, I'm leaning towards the latter.

The core of this (and more notably the previous 7777 point PR) is a gross misunderstanding of what a typical shaft miner does in a round, and why the station runs out of ore.

It's pointless and every PR raising the price of skeleton key is made up by people who don't know how mining works, what the loot is, how often tendrils are outright ignored, and don't realize that station runs out of mats primarily because miners die or do antag objectives. You can raise the price all you want, it's not going to magically make miners bring more ore, I can absolutely assure you of that.

Saying this as a miner main that doesn't really go after tendrils anymore at all unless I'm really bored because the loot is not worth the effort, and I mine just to make big number go up and then cash out buying Space Cash.

**Comment by B4CKU** [comment]
bruh

**Comment by D4C-420** [comment]
Miners don't just hand back 777 points worth of ore then buy a key. They hand in all the ore they have collected and tendrils tend to be far enough from the base that you need to do a decent amount of mining to physically go to the location of the tendril to grab said tendril. Any ore issues that occur is because they are stuck using slow and boring pka's and crushers to mine. If you want to fix the issue of not having enough ore let miners buy plasma cutters for like 1500.  
Also if the price of a key should not be greater than 1000 becuase between a key and a luxury pen most people are going to buy a pen. The current price does its job of getting miners to hand in ore to afford a key and thus the loot.

**Comment by jughu** [comment]
clearly the solution is to make megafauna crates locked

**Comment by MetalClone** [comment]
Out of virtue of #66239 being a grudge PR this shouldn't be made either. Stop making these reactionary PR's in response to lunatics who make grudge PRs, they want any kind of nerf they can get to whatever they have a grudge against. If you're worried about them making relentless cheap PRs because of their grudge, they'll eventually get gitbanned after maintaners see their obsession being the reason they push code changes instead of actual good-faith. 
Also increase the price ONLY if you increase the amount of ore on lavaland, simple as. I've mined with every implement that competes at the fastest rates of mining and it's still very hard to secure the station stable amounts of bluespace crystals and diamonds. Watcher farming is immensely slow and nets literal worthless amounts of diamonds, If you went back six years ago and told someone to just farm basilisks from the cave spawners for diamonds on the asteroid people would realize how silly it is.

Lastly can we retire the stupid fucking arguement that miners quit mining ore to fight megafauna? I hear this endlessly and it's not fucking true, you need to get ore to get luxury pens, you need 1k points for per pen, and 400 for legion core stabilizers. People quit mining because it gets fucking boring, the only people who stay mining most of the shift are legion tendril hunters and random assistants. If you want to be guaranteed to live through every single megafauna fight, you generally need to mine for around 30 minutes (depending on how fast research is) to acquire enough points to fight mega fauna, and if you're like me (someone who enjoys all aspects of the mining experience) after you finish fighting megafauna, you go back to mining and collecting other things. Mining mains I see and other randos who occasionally roll miner literally quit mining because they want to: Go roleplay, get bored of clicking rocks, don't want to fight megas, don't want to tendril hunt, AND OR don't want to ruin search. The biggest reason they usually leave is because there is barely any roleplay on lavaland to begin with, and high-pace 'war' rounds with big antags usually happen so fast you don't have a chance to acquire a 'meta' mining build, you usually can acquire 1 to 2 pieces of megafauna loot and maybe some tendril loot before the antagonists start gaining traction. The people who can fight megafauna without luxury pens and legion cores, and farm necropolis tendrils are very very rare, and it's even rarer if they get to utilize this skill in combat or killing on station. I can name only 2 people who can fight ash drakes without luxury pens (I would include myself but lag makes it so I can't risk it otherwise I can instant die to the lava minigame during a random lag spike). I can't name a single person who consistently fights bubblegum, colossus, without luxury pens, and I can't name a single person who can beat them with luxury pens and a proto-kinetic crusher and do it consistently and not die (effectively removing their ability to use their loot for killing on station later). The hierophant club has been nerfed like 4 times, and it's debatable one of the weaker get-out-of-jail-free-cards available on station, the blood-drunk miner's cleaving saw is literally weaker than a combat knife (and with all of the megafauna glitches it's harder to get safely than most stronger weapons) and those are the only two, easy to consistently beat, mega fauna that give combat loot. Lastly, the 'meta' mining builds are often impossible to complete, there are countless rounds on lavaland I spent killing and plundering only to receive horrible RNG and get the same duplicates over and over, or the many more shifts I play roundstart and there is barely any GPS signals at all. The only guaranteed spawns are the mega legion, and the hierophant (which has an extremely low chance of not spawning btw, and if that's unintended, it's a bug that still exists in the game). Everything else can and consistently doesn't show up. A fully stacked lavaland with muliple mega fauna, and over 10 necropolis tendrils is even rarer than any of that shit spawning together.
If anyone has good-faith intentions (no one who typically makes lavaland PRs is usually doing it in good-faith: i.e. the author of #66239) then figure out a way to incorporate it into roleplay more, and involve it in other departments more, trying to remove or delay acquirement of lavaland loot is usually just what drives people away from mining. A great example is how no one gave a shit about mining with the asteroid days, and in fact the best thing about the mining asteroid was the fact it had a fairly pleasant station there to escape to. The few bits of loot the asteroid had were pleasantries but ultimately 1 person usually got them all due to them not being RNG (no one cared about the abandoned crates or that evil medi-bot) so most people just quit mining after 30 minutes anyways.

**Comment by MrDoomBringer** [comment]
jesus christ just make megafauna crates drop ore too lol

**Comment by wesoda25** [comment]
Someone probably already said this in this series of PRs but you have to wonder whether the problem with miners not mining is because they choose not to, or are dead. It's very easy to say the former and just bump up prices but that's probably not all there is to it, personally I think 777 is pretty fair.

**Comment by rasepretep** [comment]
I will say again, that raising the price of skeleton keys by any amount won't have any sort of measurable impact on amount of materials gathered.

If you truly, in good faith, want miners to bring back more ore, double the amount of ore spawn on Lavaland. Problem solved.

**Comment by LemonInTheDark** [comment]
I cannot tell you how depressing it is that mining points are being used as what functionally amounts to a material gate on megafauna.

It's the most hacky fucking taped up idea in this whole codebase. By rights mining points shouldn't need to exist, but even if they do trying to use them to force people to mine materials is depressing.
JUST MAKE MINING MATERIALS FUN 4HEAD. Or, better yet, make materials matter, and use them for this.
It's fucking engineering points tier. If people aren't doing something, it's because it's not fun, or not tied into the other thing they want to do.

Even if you can't make mining fun, tying it into megafauna hunting or keys or whatever via points is the most hacked up shit man. It's just admitting that nothing else could work as a proper balance. Makes me sad.

God.

**Comment by davethwave** [comment]
There had been several discussions in forum about various options on ways to make lavaland mining more fun. But mostly the biggest obstacle was the fact of how big of a project it would be with so little of a chance of it ever being merged. 
   One of the ones that stood out to me was the whole thing being done via remotely piloting mechs that has passed around some of the more higher hiarchy in tgstation a year or so back or the boulder one where it would use a multi step machine factory like setup geared to get the most out of them kind of like mekanism.
   Honestly at this point we could just add some mining encounters to the cargo adventure drone as well as making them less hidden away then just most of the problems would be gone in terms of dead/loot hunting miners at least on the station side of things as long as they are able to provide a small amount of the rarer ores.

**Comment by RegJackson** [comment]
> can i also ask before we do this song and dance again if there is actually a problem of miners not bringing back enough ore

Don't quote me on this, it's entirely anecdotal, but I'm so rarely in this situation besides under a few circumstances, those being that

1. All/Most of the miners are dead or busy being antags
2. I just don't have enough diamonds/bluespace crystals (Which is okay, diamonds are supposed to be rare and that's fine.)
3. There's a massive engineering project like an entirely new room or something.
4. Some miner forgot to click sand and we ran out of glass.

Materials are mostly in so much abundance that you can print whatever you want as long as it doesn't require any of the 3 rarest materials on lavaland (Bananium, Bluespace, Diamond) so I'm genuinely unsure what all these PRs are talking about. I'd be interested to hear everyone else's experiences.

**Comment by Tastyfish** [comment]
The loot chests are barely worth their cost already. "oh wow gee I risked my life, spent 10 minutes dragging this crate around, and spent way more money than a luxary medipen, to get a canoe."

**Comment by optimumtact** [comment]
hey now, that canoe is lava proof

**Comment by Tastyfish** [comment]
> hey now, that canoe is lava proof

True, true

**Comment by LemonInTheDark** [comment]
That means it won't burn when you try and use it. you will, but by god it won't

**Comment by Shadowflame909** [comment]
Due to the great coder tendril rebalancing, lack of on station viability, and middling to decent effectiveness of lavaland tendril loot. I believe that fighting 4 watchers, goliaths, legions, eye beam guys, and charging lobsters is enough of an incentive to collect ores.

I already have to buy survival pens. Such an overnerf for an already balanced thing like this would make it safer to avoid the tendrils entirely! The loot just isn't worth getting hugged by a lobster

**Comment by Cheshify** [comment]
I think all of this is pointing to the bigger issue of lavaland sucking for game balance. Not entirely related, but I think it's best to take into consideration if we're gating gamer gear behind MORE meaningless points.

**Comment by bob-b-b** [comment]
An easy bandaid fix would be to start miners off with an advanced ore scanner, since i barely see anyone buy one and it makes a big difference when your trying to find ore to mine/dig paths across lavaland through big ore clusters. 

Another option is to increase ore density and give ore rocks a different visual look like on icebox, make all mobs drop a couple random ores

A price increase on keys won't really significantly increase ores mined, since as others have said the tendril loot is pretty lackluster

**Comment by StoneMason** [comment]
mining points inflation

**Comment by Fikou** [comment]
i mean like whats the point of increasing it to whatever arbitrary number. what doesnt work in the current implementation.
you spend a chunk of money for a random item that can be trash, decent, good even. you can spend more money to just get good items, like luxury pens, advanced scanners, the modsuits etc. i dont see why we need to make the random cost more than the good

**Comment by Shadowflame909** [comment]
When you do double the mining prior and fight off 4 goliaths only to get a cursed heart 💀

**Comment by SgtHunk** [comment]
> you spend a chunk of money for a random item that can be trash, decent, good even.

That's how unboxing works, you can't assure the quality of the item and if you'll be at a win/loss.
Though that does bring to mind a way for miners to at least know what they'll be getting/how good it'll be.

I thank everyone who put their feedback. This, at the end, does seem like a problem with the miner gameplay loop.
Turns out, clicking on rocks mindlessly (hoping there's no gibtonite) until you get  plasma cutters is not engaging gameplay and miners enjoy fighting megafauna because it's a challenge with a good risk/reward ratio that delays the boring gameplay.
I say this as someone who has miner on their 2nd most played job and found mining more of a break between fauna fights than the main point of the job.

I'll leave this PR up for a few more hours if there's any more discussion about what the issue with the miner gameplay is and/or if any maintainer wants to put in their opinion.

**Comment by ilikesaying** [comment]
image

**Comment by rasepretep** [comment]
> image

30 minutes real time, 25ms ping, lucky enough to have two of the three storms that hit be mild so mining can continue during them, advanced plasma cutter + bluespace satchel + night vision mesons. Nonstop mining, not even stopping to fight simple fauna. On top of that, last miner alive and mined out about 3/4 of Lavaland.

All this tells you the approximate maximum you can achieve in ideal conditions. It is in no way achievable under normal round conditions, and has zero value to the discussion. I said that multiple times on discord when it was brought up.

**Comment by Pickle-Coding** [comment]
There can be more than 150,000 points of materials to mine from lavaland, if you exclude mob farming and other weird stuff. Honestly 7,777 points doesn't seem like a lot to me, but that isn't the point.

Buying keys from vendors just doesn't thematically make sense. It probably would be a little better if they were a rare chance to drop from ores, instead of being bought from vendors. Being able to buy the healing crap and other stuff from vendors is probably enough incentive for them to bring the ores to ORM.

**Comment by rasepretep** [comment]
Keys dropping from ore makes even less sense.

On top of that, and I apparently have to keep repeating this: The only times the station station ever complains about there not being enough ore is when miners are all dead, or busy doing antag objectives. Adjusting the price of keys will have zero effect on the amount of ore mined.

Once. Again. If you want more ore in the ORM on Lavaland, double the amount of ore spawned. PRs like this are poorly thought out and work on incorrect assumptions by people not familiar with typical Shaft Miner rounds. I'm saying this as a miner main with 700+ hours on Shaft Miner.

**Comment by MrDoomBringer** [comment]
> The only times the station station ever complains about there not being enough ore is when miners are all dead,

Have you ever thought that maybe the reason miners keep dying each round is because they fight megafauna instead of mining because mining is fucking mind-numbingly boring? Why would increasing the number of ore make mining any more interesting?

**Comment by CCC23** [comment]
If you increase the price of the key you should probably add the icebox mining rewards to the tendril chests to compensate. Giving lavaland the icebox treatment of lower levels with more ores could incentivize more mining. The fix to miners not getting ores isn’t increased prices. It’s higher density and more things to do. On lavaland a miner can do all the interesting things in about 30 minutes, but on icebox you can mine for about 1 hour due to it having more things and land to explore.

**Comment by Shadowflame909** [comment]
What if lavaland fauna dropped ores like the grubhog and watcher but all of them

PVE Minecraft style

**Comment by rasepretep** [comment]
> Have you ever thought that maybe the reason miners keep dying each round is because they fight megafauna instead of mining because mining is fucking mind-numbingly boring? Why would increasing the number of ore make mining any more interesting?
> 
> Not saying increasing ores wouldn't help a little, but it ignores the key issues at play here.

First off, not all miners die to megafauna. Some just die to other things includiing accidents and lag, especially if they're new. Someone checking up on miners and going down to recover them is extremely rare. The amount of times I stepped on a cactus in front of two goliaths or lagged into lava right next to the mining base and never got found is staggering. Miners have a lot of death coming after them.

Second, mining IS mindnumbingly boring. I play miner to relax and even I get bored after an hour of making big number go up. Increasing amount of ore wouldn't make mining interesting at all, but since the very original complaint was that "miners don't mine enough ore because skeleton keys are too cheap", it's a better solution than increasing key cost, because key cost is not the cause of low ore amounts. More ore available may mean that miners pick up more ore per trip. It doesn't address the issue of mining being boring because that's not the why these weird PRs exist.

Third: Mining needs a rework. Badly. Mining needs a rework so hard that it's not even funny anymore. Shaft Miner has become Dark Souls with a mining mechanic. Personally I like it, because I enjoy the danger of having to dodge fauna while I'm getting ore for the station and the everpresent threat of getting round removed at the slightest mistake, but bottom line it's still Dark Souls with a mining minigame that nobody likes.

> If you increase the price of the key you should probably add the icebox mining rewards to the tendril chests to compensate.

Oh god please don't. There's already a mere 1:20 chance to get what you're looking for, and bloating it with all the literal garbage from Icebox would make it even worse.

> What if lavaland fauna dropped ores like the grubhog and watcher but all of them

Again, low ore amounts are not because miners are busy fighting stuff. ORM is empty because miners died fighting stuff, or fucked off to do antag objectives. Plus unless the creatures drop like 50 titanium per kill, no miner is gonna bother farming them, because it's much faster to just mine.

As a sidenote, the very VERY original complaint about miners and skeleton keys that spawned the ridiculout 7777 points for skeleton key, was some rambling about "miner meta loadout from tendril loot". It was a ridiculous kneejerk / grudge PR but unfortunately it spawned this entire weird discussion.

And on a final note, miners do mine and skeleton keys is almost the last thing they buy. Miners spend their points primarily on luxpens, stabilizing serums, MODsuits and sometimes advanced scanner or the new kheiral links.

This entire series of discussions is getting tiresome and once again: will achieve nothing in terms of dealing with the instances of station being low on ore. If it even is a problem, I would really like to see how many rounds out of a week the station had no ore and what happened.

**Comment by Learner10** [comment]
Deep Rock Galactic has mining missions of various sorts and various threat. You're rewarded for completing those missions through currency, and sometimes materials. They are randomly generated and never quite the exact same.

Frankly, I don't see why the approach to all of this hasn't been to look into a way to provide some dynamic mission events for miners to do that can reward them with both materials and points as a means to justify raising the prices. It'd help keep lavaland from remaining this static flat slice of Swiss cheese that you're constantly drilling holes into. It'd inject a lot more creativity into lavaland's design through new missions, mission rewards, code components, and enemy types... and it'd help cut down on what is ultimately in the end a limited pool of material to be found and used in a round (even if there is a great deal should you desperately start looking).

In short... Lavaland's current status is reasonably well off (or at least understood), so the best way to change it would be to have new content to balance around.

**Comment by Kylerace** [comment]
im not convinced mining key prices need to be raised and agree with lemon on the matter
